### DISCUSSION TITLE
Robots.txt File

### DISCUSSION THREAD

**Discussion by aarajbhattarai** [vercel/next.js#53604] [App Router]
Hello, I get in 500 Internal Server Error when I try to access the robots.txt file. I have followed the file-based convention for Metadata Generation. From the same directory, the favicon is being fetched. The file robot.txt is generated when the building the build package. The screenshots are below

**Folder Structure**
<img width="216" alt="Screenshot 2023-08-05 at 2 39 06 PM" src="https://github.com/vercel/next.js/assets/53218772/19e08b5b-c456-484b-85a1-2eefd611f80a">

**Build Log**
<img width="683" alt="Screenshot 2023-08-05 at 2 40 48 PM" src="https://github.com/vercel/next.js/assets/53218772/e4f3ccc3-b8db-4c2e-ad9c-72a3b1d94127">

**Comment by Nadeera3784** [comment]
Hi, @aarajbhattarai 
After the production build, is the robots.txt file available in the public folder?
